.jfrog.taskcat.yml

```diff
@@ -3,14 +3,14 @@ project:
   owner: quickstart-eng@amazon.com
   s3_object_acl: private
   s3_regional_buckets: True
-  s3_bucket: tcat-34c17b83dcbd547393a86d8053a8d298 # commercial accounts (also for GovCloud)
-  #s3_bucket: tcat-a3e80b6745b2547da1c745b16adf2a66    # aws-seller accounts (MarketPlace)
+  s3_bucket: tcat-qs-ec2 # commercial accounts (also for GovCloud)
+  #s3_bucket: tcat-qs-ec2    #aws-seller accounts (MarketPlace)
```

Comment:
Not sure if we can/should share same S3 bucket between account. Should be fine as long as the sub-directories are unique for each account.

Reply:
yeah I thought of using same s3 bucket for all but sometimes when testing if it fails we make changes and run another while the existing stack is getting removed.Also taskcat uploads with the same name by default

Reply:
Ok fine for now.